README.md

```diff
@@ -126,6 +127,17 @@ cd "$(basename "${REPO}" .git)"
 git status
 ```
 
+### Special repository checks
+
+For some repositories some extra manual validation needs to be performed before the release branch is cut:
+
+#### knative/client
+
+* Check that in [version.go](https://github.com/knative/client/blob/main/pkg/kn/commands/version/version.go) the variavle [apiVersions](https://github.com/knative/client/blob/main/pkg/kn/commands/version/version.go#L32) points to the versions of `knative-serving` and `knative-eventing` that is about to be released. You can verify that that the version is picked up properly by calling `./hack/build.sh` from within the client repository.
```

Comment:
nit: variavle => variable?

Reply:
nit: 'that that the version' => 'that the version"

Reply:
Just checking if it's enough to check that particular version (in github version.go) is enough, or does one need to manually checkout the repo and run the command? If the second step is optional, then maybe omit it (or make it optional like below)?

Reply:
Good point. It should be optional, we have a unit test that checks the proper output.